Jenkinsfile

```diff
@@ -128,10 +128,12 @@ spec:
   - name: jnlp
     env:
       - name: JNLP_PROTOCOL_OPTS
-        value: "-XshowSettings:vm -Xmx2048m -Dsun.zip.disableMemoryMapping=true -Dorg.jenkinsci.remoting.engine.JnlpProtocol3.disabled=true"
+        value: "-XshowSettings:vm -Xmx1024m -Dsun.zip.disableMemoryMapping=true -Dorg.jenkinsci.remoting.engine.JnlpProtocol3.disabled=true"
```

Comment:
I think that previously we were using a 512mb max Java heap size for JNLP and that was when we occasionally hit out of memory failures when doing git operations.  I have thought of using a lower heap size and memory limit, which we are doing here.  

New max Java heap size for jnlp is 1024mb and the container/vm max memory setting is 2.5gb (with initial memory size of 1.5gb).

Reply:
https://kubernetes.io/docs/tasks/administer-cluster/manage-resources/ describes the Kubernetes memory settings options.